.github/CODEOWNERS

```diff
@@ -35,10 +35,10 @@ browser/ui/webui/brave_rewards_ui.cc @brave/rewards-client
 browser/ui/webui/brave_rewards_ui.h @brave/rewards-client
 components/brave_rewards/ @brave/rewards-client
 components/services/bat_rewards/ @brave/rewards-client
-components/brave_rewards/core/common/signer.cc @brave/sec-team
-components/brave_rewards/core/common/signer.h @brave/sec-team
-components/brave_rewards/core/common/request_signer.cc @brave/sec-team
-components/brave_rewards/core/common/request_signer.h @brave/sec-team
+components/brave_rewards/core/internal/common/signer.cc @brave/sec-team
```

Comment:
you still have `common` here which is a process directory, but not sure what makes sense here instead

Reply:
Raised https://github.com/brave/brave-browser/issues/43304 to fix the directory structure

Reply:
That folder contains utility functions and classes for use by the "engine". The folder could be renamed to "util" or "helpers" or something similar.

Reply:
either of those options are fine, I just don't want to confuse the use of `common` due to its special meaning in chromium